### PR TITLE
Reduce debug noise

### PR DESCRIPTION
**Pull request by m-kuhn**
We have a lot of debug noise currently on the "warnings" channel.
This makes it almost impossible to find warnings which are emitted for a good reason. This is an attempt to reduce that and slowly move towards a warning-free situation.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-636-f98469b6f19eba493b3a5608725ce499136f2c4a-armv7-X.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-636-f98469b6f19eba493b3a5608725ce499136f2c4a-x86-X.apk) (Android 5: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-636-f98469b6f19eba493b3a5608725ce499136f2c4a-armv7-5.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-636-f98469b6f19eba493b3a5608725ce499136f2c4a-x86-5.apk))

File: src/qml/DashBoard.qml [hunk 1]
```diff
@@ -45,7 +44,7 @@ Controls.Pane {
   onShowMenu: mainMenu.__popup(Qt.rect(menuButton.x + 2 * menuButton.width, menuButton.y, mainMenu.__popupGeometry.width, mainMenu.__popupGeometry.height), 0, 0)
 
   onCurrentLayerChanged: {
-    if ( currentLayer.readOnly && stateMachine.state == "digitize" )
+    if ( currentLayer && currentLayer.readOnly && stateMachine.state == "digitize" )
```
Comment (signedav) [inline src/qml/DashBoard.qml]: I suppose if you did it, then it's needed. But why?

Comment (mbernasocchi) [reply src/qml/DashBoard.qml]: isn't it to avoid complaints that the layer is read-only when there is no layer selected?

Comment (signedav) [reply src/qml/DashBoard.qml]: So apparently readOnly of "no layer" is true? I'm just curious...

Comment (signedav) [reply src/qml/DashBoard.qml]: It's because there is a warning that it cannot access the property readOnly when undefined... Fine. It's clear now.

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-636-5d8b433ae7d1c6dd0b6409390f7dcfbb2e2ba052-armv7-X.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-636-5d8b433ae7d1c6dd0b6409390f7dcfbb2e2ba052-x86-X.apk) (Android 5: [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-636-5d8b433ae7d1c6dd0b6409390f7dcfbb2e2ba052-armv7-5.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-636-5d8b433ae7d1c6dd0b6409390f7dcfbb2e2ba052-x86-5.apk))

**Comment by signedav** [comment]
Thanks @m-kuhn

**Pull request merged**